.github/workflows/release_canary.yml

```diff
@@ -11,6 +11,10 @@ on:
         type: number
         required: false
         default: 16
+      useGPR:
```

Comment:
For consistency with other env vars can we use underscores for the inputs please? Sorry, just noticed this now.

`use_gpr` instead of `useGPR`

While we're here, also rename `node-version` to `node_version`.